themes/index.js

```diff
@@ -221,6 +221,12 @@ const themes = {
     text_color: "ffffff",
     bg_color: "20232a",
   },
+  pastel: {
+    title_color: "c3aed6",
+    icon_color: "efbbcf",
+    text_color: "ffd5cd",
+    bg_color: "8675a9"
```

Comment:
```suggestion
    bg_color: "6D4FAC"
```

make the background a little darker, so that contrast is better:

![image](https://user-images.githubusercontent.com/34424160/92796614-3efcd100-f3a9-11ea-8c04-97da430d2989.png)